src/google/protobuf/compiler/cpp/helpers.cc

```diff
@@ -41,9 +41,9 @@
 #include <map>
 #include <memory>
 #include <queue>
-#include <unordered_set>
 #include <vector>
 
+#include "absl/container/flat_hash_set.h"
```

Comment:
feels weird to put it at this point in the include list, but sure

Reply:
I was aiming for where it won't get reordered in google3.  "third_party/absl" < "third_party/protobuf" :)